Tests/scripts/lock_cloud_machines.py

```diff
@@ -361,9 +361,8 @@ def main():
     # remove build from queue
     remove_file(storage_bucket, file_path=f'{options.gcs_locks_path}/{QUEUE_REPO}/{options.ci_job_id}')
 
-    # the output need to be improved if we wont to support locking for multiply machines.
     with open(options.response_machine, "w") as f:
-        f.write(f"export CLOUD_CHOSEN_MACHINE_ID={lock_machine_list[0]}")
+        f.write(f"export CLOUD_CHOSEN_MACHINE_IDS={','.join(lock_machine_list)}")
 
```

Comment:
**this is the reason to the major change in all files. i've added support to locking more than one machine per a build pipeline.**

so i changed the variable from `CLOUD_CHOSEN_MACHINE_ID` to `CLOUD_CHOSEN_MACHINE_IDS` to indicate it's a multiple values, that's the comment that existed before that it's not supported, that's why the comment was removed.